src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -8,6 +8,10 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number < 0) {
+            throw new IllegalArgumentException("Expected non-negative number, got "
+                    + number + " ?");
```

Comment:
```suggestion
                    + number);
```
`...где вместо вопросика будет подставлено фактически переданное в метод число.` :)